docs/tutorials/simple-extensions.md

```diff
@@ -356,5 +356,5 @@ scene.setTile(7, img`
 Congratulations, your platformer is complete! See if you can get to the wall at the end of the level.
 
 ```package
-corgio
-```
\ No newline at end of file
+corgio=github:jwunderl/pxt-corgio#master
```

Comment:
Before merging, this ought to be changed back to just reference the version in `libs/`